WcaOnRails/config/initializers/translations.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+if defined?(Rails::Server) # See: https://stackoverflow.com/a/44441168
+  Rails.configuration.after_initialize do
+    modification_timestamp = Timestamp.find_or_create_by!(name: 'en_translation_modification')
+    latest_modification_date = DateTime.parse(`git log -1 --format='%ai' #{Rails.root.to_s}/config/locales/en.yml`)
+    if modification_timestamp.date != latest_modification_date
+      TranslatorsMailer.notify_translators_of_changes.deliver_later unless modification_timestamp.date.nil?
```

Comment:
Just making sure I understand this unless statement... The idea is to not send the email the very first time, and instead just update the timestamp in our database? After that, we'll send the email?